src/utils/formHelperFunctions.ts

```diff
@@ -504,3 +506,34 @@ export function isStepTerminal(step: any) {
 
   return !hasNext;
 }
+
+export function saveUrlParams(
+  updateFieldValues: (newFieldValues: any, rerender?: boolean) => boolean,
+  client: Client
+) {
+  const params = new URLSearchParams(featheryWindow().location.search);
+  const hiddenFields: any = {};
+  params.forEach((value, key) => {
+    if (['feathery_1', 'feathery_2', '_slug'].includes(key)) return;
```

Comment:
i think you can ignore feathery_1 and feathery-2 now, it's been long enough and i've removed that handling logic form hosted forms too

Reply:
I wanted to make sure that the stytch query params didn't get saved